guides/hack/10-types/79-type-aliases.md

```diff
@@ -33,8 +38,9 @@ function distance(Point $p1, Point $p2): float {
 }
 ```
 
-Being in the same source file as the alias definition, the functions `create_Point` and `distance` have direct access to the integer
-fields in any `Point`'s tuple.  However, any file that includes this file does not.
+## `Type` v. `Newtype` | (Transparent v. Opaque)
+
+Looking at the earlier example, being in the same source file as the alias definition, the functions `create_Point` and `distance` have direct access to the `float` fields in any `Point`'s tuple.  However, any file that includes this file does not.
```

Comment:
include/require are very rarely used nowadays and pretty discouraged; perhaps something along the lines of 'no other file' instead?